pkg/inspect/inspect.go

```diff
@@ -44,23 +44,23 @@ type HostConfig struct {
 	ShmSize              int64                       `json:"ShmSize"`
 	Runtime              string                      `json:"Runtime"`
 	ConsoleSize          *specs.Box                  `json:"ConsoleSize"`
-	Isolation            string                      `json:"Isolation"` //TODO
-	CPUShares            *uint64                     `json:"CPUSShares"`
+	Isolation            string                      `json:"Isolation"`  //TODO
+	CPUShares            *uint64                     `json:"CpusShares"` //nolint
 	Memory               int64                       `json:"Memory"`
-	NanoCPUs             int                         `json:"NanoCPUs"` //check type, TODO
+	NanoCPUs             int                         `json:"NanoCpuss"` //nolint
 	CgroupParent         string                      `json:"CgroupParent"`
 	BlkioWeight          *uint16                     `json:"BlkioWeight"`
 	BlkioWeightDevice    []specs.LinuxWeightDevice   `json:"BlkioWeightDevice"`
 	BlkioDeviceReadBps   []specs.LinuxThrottleDevice `json:"BlkioDeviceReadBps"`
 	BlkioDeviceWriteBps  []specs.LinuxThrottleDevice `json:"BlkioDeviceWriteBps"`
 	BlkioDeviceReadIOps  []specs.LinuxThrottleDevice `json:"BlkioDeviceReadIOps"`
 	BlkioDeviceWriteIOps []specs.LinuxThrottleDevice `json:"BlkioDeviceWriteIOps"`
-	CPUPeriod            *uint64                     `json:"CPUPeriod"`
-	CPUQuota             *int64                      `json:"CPUQuota"`
-	CPURealtimePeriod    *uint64                     `json:"CPURealtimePeriod"`
-	CPURealtimeRuntime   *int64                      `json:"CPURealtimeRuntime"`
-	CPUSetCPUs           string                      `json:"CPUSetCPUs"`
-	CPUSetMems           string                      `json:"CPUSetMems"`
+	CPUPeriod            *uint64                     `json:"CpuPeriod"`          //nolint
```

Comment:
Do we have to nolint these given it's only the JSON tag that's not properly named? I think we don't